.github/actions/bazel/action.yaml

```diff
@@ -73,46 +73,42 @@ runs:
           ${{ steps.metrics-tmpdir.outputs.dir }}/bazel-bep-*.pb.gpg
           ${{ steps.metrics-tmpdir.outputs.dir }}/profile-*.json
 
-
-      # Remove unnecessary entries from the JSON execlogs and create a CSV out of them
-    - name: Clean up execlogs
+    - name: Build execution_log_compact_to_csv
```

Comment:
is there a reason for building this separately and then referring to it by path instead of doing `bazel run`?

Reply:
I wanted the bazel invocation to get all the required arguments that we need on CI. If I would use a `bazel run` I have to repeat all those arguments. But I'm not married to this approach so I'm open to change it.

Reply:
I forgot this was inside the bazel action itself! I'm actually a bit uneasy running a bazel build inside the bazel action wrapper itself. Thinking out loud, I'm wondering if it would make sense to extract this into another repo and provide a statically built executable that we can just download here

Reply:
I'm not sure about the suggestion about fetching a static binary from a separate repo. That seems to complicate things and I would rather avoid depending on even more binary inputs into our builds.

I switched to just running the script using `bazel run`. Seems much simpler indeed.